content/introduction.html

```diff
@@ -1,15 +1,18 @@
 <h1> About the program </h1>
 <p class="full">
-  The University of Waterloo’s Software Engineering (SE) is an interdisciplinary
+  The Software Engineering (SE) at the University of Waterloo is an interdisciplinary
   program supported by both the Faculty of Mathematics and the Faculty of
-  Engineering. For 5 years, all 141 graduating students of the class of 2018 spent 8
-  semesters in school with similar schedules as well as 6 co-ops.
+  Engineering. For 5 years, all 92 graduating students of the class of 2021 (SE'XXI)
+  spent 8 semesters in school with similar schedules as well as 6 co-ops.
```

Comment:
should we add a sentence along the lines of

```
Undergraduates in the Software Engineering program are awarded a Bachelor's of Software
Engineering (B. SE.) at the end of their 4 2/3 year journey.
```

to distinguish us from other programs? (McGill awards a bachelor of science i think, and some other places a bachelor of arts)

Reply:
Yes! I also added the term "Baccalaureate"